### ISSUE TITLE
credsStore desktop docker login No response POST /registry/credstore-updated

### ISSUE THREAD

**Issue by tedli** [docker/for-win#11472]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: `C951C974-49B5-48A6-B464-7631975236BE/20210617132938`

### Actual behavior

```
docker login
Authenticating with existing credentials...
time="2021-06-17T21:25:23+08:00" level=error msg="(a23369cc) d6acd1b3-CredentialHelperPKG C<-S No response POST /registry/credstore-updated (1.0127257s): Post \"http://ipc/registry/credstore-updated\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)[[STACK]]"
time="2021-06-17T21:25:24+08:00" level=error msg="(b35baa75) d6acd1b3-CredentialHelperPKG C<-S No response POST /registry/credstore-updated (1.0114488s): Post \"http://ipc/registry/credstore-updated\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)[[STACK]]"
```

### Expected behavior

login success

### Information

  - Windows Version:
  - Docker Desktop Version:
  - WSL2 or Hyper-V backend?  
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM:

### Steps to reproduce the behavior

  1. ...
  2. ...

sorry i don't know what caused this. i can login long time ago, when i need to login to a new registry, i found this, i nearly did nothing since i can login, because everything works as expect, until today, login failed.

i did factory reset, uninstall, reinstall, didn't solve

**Comment by yamakca** [comment]
same behavior an error

**Comment by taliesins** [comment]
This is what I used to login despite "credStore" being set to "desktop".

```
$containerRegistryName = '<ContainerRegistryName>.azurecr.io'
$result = az acr login -n $containerRegistryName --expose-token
$resultJson = ConvertFrom-Json ($result -Join '')
docker login $containerRegistryName --username 00000000-0000-0000-0000-000000000000 --password $($resultJson.accesstoken)
```

**Comment by StefanScherer** [comment]
Thanks @tedli for the report. We will ship a fix in the next release.

**Comment by StefanScherer** [comment]
We have just released Docker Desktop 3.5.0 which contains a fix for this issue. I'll close this ticket now, but feel free to comment and give us feedback. If there is still a problem, please open a new ticket.

**Comment by tedli** [comment]
@StefanScherer 
Thanks. It solved.
![image](https://user-images.githubusercontent.com/3324663/123190015-66a0ce00-d4d1-11eb-9c70-4bc0a4518982.png)
